doc/source/actions.rst

```diff
@@ -254,6 +254,17 @@ The ``merge`` action merges the pull request into its base branch. The
        * GPG signed commits will lost their signatures.
        * Also see: :ref:`faq strict rebase`
 
+When the Pull Request is merged, the default merge/squash commit message is the
+Github one. But you can override it by adding a new section into the Pull
+Request body starting with the Markdown title "Commit message", followed by the
+commit message, for example::
+
+    ## Commit message
+    My wanted commit title
+
+    The whole commit message will finish at the body end or next line starting
+    by a markdown title.
```

Comment:
The whole commit message finishes at the end of the pull request body or before a new Markdown title.

(avoid future tense in technical documentation)